cocoa/src/toga_cocoa/window.py

```diff
@@ -266,3 +266,18 @@ def cocoa_windowShouldClose(self):
 
     def close(self):
         self.native.close()
+
+    def get_current_screen(self):
+        screen_native = self.native.screen
+        for screen in self.interface._app.screens:
+            if screen_native == screen._impl.native:
+                return screen._impl
+
+    # def set_current_screen(self, app_screen):
+    #     self.native.setFrameOrigin(app_screen._impl.native.visibleFrame.origin)
```

Comment:
There's no need to keep code commented out; if we're not using them, we don't need them; and if the implementation is interesting for some reason, it's preserved in git history.